backend/python/app/tests/services/test_story_service.py

```diff
@@ -67,9 +72,44 @@ def test_get_stories_available_for_translation():
     pass
 
 
-def test_create_translation():
-    pass
+def test_create_translation(app, db, services):
+    alphanumeric = 'abcdefghijklmnopqrstuvwxyz0123456789'
+
+    translator_authId = ''.join(random.choice(alphanumeric) for i in range(10))
+    translator_obj = db_session_add_commit_obj(
+        db, UserAll(first_name="Translatorfirst", last_name="Translatorlast", email=f"{translator_authId}", is_deleted=False, auth_id=translator_authId)
+    )
+
+    reviewer_authId = ''.join(random.choice(alphanumeric) for i in range(10))
+    reviewer_obj = db_session_add_commit_obj(
+        db, UserAll(first_name="Reviewerfirst", last_name="Reviewerlast", email=f"{reviewer_authId}", is_deleted=False, auth_id=reviewer_authId)
+    )
+
+    story = StoryRequestDTO(
+        title="Title",
+        description="Description",
+        youtube_link="",
+        level=1,
+        translated_languages=[],
+    )
+    content = ["Story content 1.", "Story content 2.", "Story content 3."]
+    story_resp = services["story"].create_story(story, content)
+    story_obj = Story.query.get(story_resp.id)
+
+    story_translation = StoryTranslationRequestDTO(
+        story_id=story_obj.id,
+        language="English",
+        stage="TRANSLATE",
+        translator_id =translator_obj.id,
+        reviewer_id=reviewer_obj.id,
+    )
+    story_translation_resp = services["story"].create_translation(story_translation)
+    story_translation_obj = StoryTranslation.query.get(story_translation_resp.id)
+    assert story_translation_resp == story_translation_obj
 
+    get_story_translation_resp = services["story"].get_story_translation(story_translation_obj.id)
+    content_objs = StoryContent.query.filter_by(story_id = story_resp.id).all()
+    assert len(content_objs) == get_story_translation_resp["num_content_lines"]
```

Comment:
Is this the correct way to compare content lines?

Reply:
this is a good check! additionally you should check the actual lines on the response body, eg
`    assert len(content_objs) == len(get_story_translation_resp["translationContents"])`